app/routes/goal_routes.py

```diff
@@ -1 +1,110 @@
-from flask import Blueprint
\ No newline at end of file
+from flask import Blueprint, request, abort, make_response
+from app.models.goal import Goal
+from app.models.task import Task
+from app.db import db
+from app.routes.route_utilities import validate_model
+
+bp = Blueprint("goal_bp", __name__, url_prefix="/goals")
+
+@bp.post("")
+def create_goal():
+    request_body = request.get_json()
+
+    if "title" not in request_body:
+        return {"details": "Invalid data"}, 400
+
+    title = request_body["title"]
+
+    new_goal = Goal(title=title)
+    db.session.add(new_goal)
+    db.session.commit()
+
+    return {"goal": new_goal.to_dict()}, 201
```

Comment:
I see that we have a `create_model` function in our utilities function, is there a reason as to why we didn't utilize it here?